src/components/Consent/createConsentRequest.js

```diff
@@ -17,7 +17,7 @@ export default ({ payload, datastreamIdOverride }) => {
     payload,
     datastreamIdOverride,
     getAction() {
-      return "privacy/set-consent";
+      return "consent/set-consent";
```

Comment:
This is part of the Konductor url for `set-consent` commands and needs to remain as `"privacy/set-consent"`. As is, `set-consent` commands return a 404 response.

Reply:
Good catch.